developers/prerequisites/osgitasks.md

```diff
@@ -119,13 +119,17 @@ interface `ElectricityConsumer` with methods:
 
 1. Implement the [*ManagedService*][ManagedService] interface in the `org.openhab.training.electricity.tv.events` bundle and:
 	- add a configuration to the `TV` class with property "autoSleep" and value a timestamp (hh:mm);
-	- modify the method that listens for events with topic "time" to stop the bundle, when the "autoSleep" property is equal to the "time" topic.
-2. Create another implementation of the `ElectricityProvider` interface (`RechargableBattery`) in an `org.openhab.training.electricity.rechargablebattery` bundle that:
+	- modify the method that listens for events with topic "time" to stop the bundle, when the "autoSleep" property is equal to the "time" topic.
+
+2. Create another implementation of the `ElectricityProvider` interface (`RechargableBattery`) in an `org.openhab.training.electricity.rechargeablebattery` bundle that:
 	- has a  finite charge (e.g `int capacity = 30`);
 	- implements the *ManagedService* interface. After this modification, it should be possible to change the charge of a battery with a `setCharge(int capacity)` method;
-	- when the battery is recharged (the configuration of the battery is changed), the consumers must be notified about the change.
 
-      Hint! Read about the [ConfigurationAdmin](https://osgi.org/javadoc/r4v42/org/osgi/service/cm/ConfigurationAdmin.html) service and how you can use it to change the configuration of a ManagedService. You might want to implement additional bundle that is using the ConfigurationAdmin service to change the configuration of the battery to perform a quick test.
+    Hint! Read about the [ConfigurationAdmin](https://osgi.org/javadoc/r4v42/org/osgi/service/cm/ConfigurationAdmin.html) service and how you can use it to change the configuration of a ManagedService. You might want to implement additional bundle that is using the ConfigurationAdmin service to change the configuration of the battery to perform a quick test.
```

Comment:
While we have the hints in the other pages in a gray box, these here does not have specific formatting. I think that it will be helpful for the reader to apply the same formatting in all the wiki pages.